From d1a909dd8075c85bf093ce0e53cb3584190031a9 Mon Sep 17 00:00:00 2001
From: Darren Etheridge <detheridge@ti.com>
Date: Tue, 25 Apr 2023 17:15:16 -0500
Subject: [PATCH 419/508] media: img: vxe-vxd: add vxe-vxd driver to
 MAINTAINERS file

Add the IMGTEC vxe-vxd driver to the maintainers file.

Signed-off-by: Darren Etheridge <detheridge@ti.com>
---
 MAINTAINERS | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 052fbbe113f7..5c509258f830 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -9990,6 +9990,14 @@ S:	Maintained
 F:	Documentation/devicetree/bindings/auxdisplay/img,ascii-lcd.yaml
 F:	drivers/auxdisplay/img-ascii-lcd.c
 
+IMGTEC VIDEO CODEC V4L2 DRIVER
+M:	Darren Etheridge <detheridge@ti.com>
+M:	Brandon Brnich <b-brnich@ti.com>
+S:	Maintained
+F:	Documentation/devicetree/bindings/media/img,d5500-vxd.yaml
+F:	Documentation/devicetree/bindings/media/img,vxe384.yaml
+F:	drivers/media/platform/img/vxe-vxd
+
 IMGTEC IR DECODER DRIVER
 S:	Orphan
 F:	drivers/media/rc/img-ir/
-- 
2.41.0

